test/FunctionalTests/Tests/DataServices/UnitTests/DesignT4UnitTests/CodeGenerationContextTest.cs

```diff
@@ -294,5 +294,72 @@ public void GetEdmxStringFromMetadataPathShouldThrowExceptionIfInputIsInvalidUri
                 Assert.AreEqual(ex.Message, "Only file, http, https schemes are supported for paths to metadata source locations.");
             }
         }
+
+        [TestMethod]
+        public void SetOneCustomHeaderCorrectlyShouldNotThrowException()
+        {
+            var codegen = new ODataT4CodeGenerator();
+            string customHeaderString = "{Authorization:Bearer bearer-token}";
+            codegen.ValidateAndSetCustomHttpHeadersFromString(customHeaderString);
+            Assert.IsNotNull(codegen.CustomHttpHeaders);
+            Assert.AreEqual(1, codegen.CustomHttpHeaders.Count);
+            Assert.AreEqual("Authorization", codegen.CustomHttpHeaders.FirstOrDefault().Key);
+            Assert.AreEqual("Bearer bearer-token", codegen.CustomHttpHeaders.FirstOrDefault().Value);
+        }
+
+        [TestMethod]
+        public void SetMultipleCustomHeadersCorrectlyShouldNotThrowException()
+        {
+            var codegen = new ODataT4CodeGenerator();
+            string customHeaderString = "{Authorization:Bearer bearer-token},{odata.continue-on-error:true}";

```

Comment:
```suggestion
            var customHeaderString = "{Authorization:Bearer bearer-token},{odata.continue-on-error:true}";
```

Reply:
@odero The style in this file seems to prefer `type` over `var`, so I have used string for consistency